profiles/mullvad-adblock-doh.mobileconfig

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	<array>
+		<dict>
+			<key>DNSSettings</key>
+			<dict>
+				<key>DNSProtocol</key>
+				<string>HTTPS</string>
+				<key>ServerAddresses</key>
+				<array>
+					<string>2a07:e340::3</string>
+					<string>194.242.2.3</string>
+				</array>
+				<key>ServerURL</key>
+				<string>https://adblock.doh.mullvad.net/dns-query</string>
+			</dict>
+			<key>PayloadDescription</key>
+			<string>Configures device to use Mullvad DNS over HTTPS</string>
+			<key>PayloadDisplayName</key>
+			<string>Mullvad DNS over HTTPS</string>
+			<key>PayloadIdentifier</key>
+			<string>com.apple.dnsSettings.managed.4A06FE90-FBD6-4603-8CEC-FD1C94BF304C</string>
+			<key>PayloadType</key>
+			<string>com.apple.dnsSettings.managed</string>
+			<key>PayloadUUID</key>
+			<string>4A06FE90-FBD6-4603-8CEC-FD1C94BF304C</string>
+			<key>PayloadVersion</key>
+			<integer>1</integer>
+			<key>ProhibitDisablement</key>
+			<false/>
+		</dict>
+	</array>
+	<key>PayloadDescription</key>
+	<string>Configures device to use Mullvad DNS over HTTPS</string>
+	<key>PayloadDisplayName</key>
+	<string>Mullvad DNS over HTTPS</string>
+	<key>PayloadIdentifier</key>
+	<string>com.sunknudsen.mullvad-dns-over-https</string>
```

Comment:
can you change this to whatever every other mobileconfig is using? makes it easier to to diffs, audits, etc

Reply:
Sure, will fix tomorrow morning.

Reply:
@paulmillr Fixed

Reply:
Nope. Payloadidentifier still differs from files in the repo.

Reply:
Btw, you could be a little less blunt when people are contributing to your project.

Reply:
@paulmillr Fixed